ckanext/canada/templates/public/package/base.html

```diff
@@ -3,9 +3,11 @@
 {% set client_lang = h.lang() %}
 
 {% block breadcrumb_content %}
-  {% set dataset = pkg.title[client_lang] %}
   {% set pkg_url = h.url(controller='package', action='read', id=pkg.id) %}
   <li>{% link_for _('Search Open Data'), named_route='dataset_search' %}</li>
-  <li>{% link_for dataset|truncate(30), controller='package', action='read', id=pkg.name %}</li>
+  {% if pkg.title_translated is defined %}
+    {% set dataset = pkg.title_translated[client_lang] %}
+    <li>{% link_for dataset|truncate(30), controller='package', action='read', id=pkg.name %}</li>
+  {% endif %}
```

Comment:
if use  `h.get_translated(pkg, 'title')`, the link http://localhost:5000/en/dataset/new will crash. 
`TypeError: string indices must be integers, not str`
because of pkg=''. How to solve this properly?